usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -119,17 +116,14 @@ def signal_handler(signal, frame):
                     .format(submission_id, award_financial_frame.shape[0]))
         logger.info('Loading File C data')
         start_time = datetime.now()
-        awards_touched = load_file_c(submission_attributes, db_cursor, award_financial_frame)
+        load_file_c(submission_attributes, db_cursor, award_financial_frame)
         logger.info('Finished loading File C data, took {}'.format(datetime.now() - start_time))
 
+        # update C to D linkage using SQL scripts
+        call_command('update_file_c_linkages')
+
         if not options['nosubawards']:
-            try:
-                start_time = datetime.now()
-                logger.info('Loading subaward data...')
-                load_subawards(submission_attributes, awards_touched, db_cursor)
-                logger.info('Finshed loading subaward data, took {}'.format(datetime.now() - start_time))
-            except Exception:
-                logger.warning("Error loading subawards for this submission")
+            logger.warning("Loading subaward during a submission load is no longer supported")
```

Comment:
@nmonga91 What is the impact on data completeness and quality if we are not attempting to load subawards any longer? 
Do we need to compensate with backfilling subawards later? Does your SQL somehow load them too? If not, when will they be loaded? 

Also, might it be better to leave this code here and just have the jenkins job stage pass in the `nosubawards` option?

Reply:
The jenkins job already passes `nosubawards` and this method doesn't load it correctly to begin with, so I just essentially "deprecated" it